drivers/ncv7356/ncv7356.c

```diff
@@ -0,0 +1,77 @@
+/*
+ * Copyright (C) 2016-2018  OTA keys S.A.
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @ingroup     drivers_ncv7356
+ * @{
+ *
+ * @file
+ * @brief       generic can transceiver implementation for ncv7356
+ *
+ * @author      Aurelien GONCE <aurelien.gonce@altran.com>
+ * @author      Vincent Dupont <vincent@otakeys.com>
+ *
+ * @}
+ */
+
+#include "ncv7356.h"
+
+int ncv7356_trx_set_mode(can_trx_t *dev, can_trx_mode_t mode)
+{
+    ncv7356_trx_t *ncv7356 = (ncv7356_trx_t *)dev;
+    int ret;
+
+    switch (mode) {
+    case TRX_NORMAL_MODE:
```

Comment:
case should be indented

Reply:
there are some places where cases are not indented, so I guess we can keep it like that.